server/test/integration/apiTest/messageClientsTest.js

```diff
@@ -0,0 +1,328 @@
+// Third-party dependencies
+const chai = require('chai')
+const chaiHttp = require('chai-http')
+const sinonChai = require('sinon-chai')
+const { afterEach, beforeEach, describe, it } = require('mocha')
+const sinon = require('sinon')
+
+// In-house dependencies
+const { factories, helpers, twilioHelpers } = require('brave-alert-lib')
+const { buttonDBFactory } = require('../../testingHelpers')
+const { server, db } = require('../../../server')
+
+chai.use(chaiHttp)
+chai.use(sinonChai)
+
+const expect = chai.expect
+const sandbox = sinon.createSandbox()
+const braveKey = helpers.getEnvVar('PA_API_KEY_PRIMARY')
+const badBraveKey = 'badbravekey'
+const message = 'Hello, world!'
+const testFromPhoneNumber = '+11234567890'
+
+// information about clients that should be contacted
+const testExpectPhoneNumbers = ['+11111111111', '+12222222222', '+13333333333', '+14444444444']
```

Comment:
This is great. Pretty complex test case that you had to set up in order to do this. Thanks for sticking with it. 

I've added some comments on other cases that could be included. When it starts getting picky like that, then you might think about moving some of this to the unit level and calling `api.messageClients` directly with a mocked `req`, `res` and `db`.